applications/nginx-registration/Dockerfile

```diff
@@ -1,17 +1,8 @@
 FROM nginx:alpine
 
-RUN apk update
```

Comment:
Mente du i commit-melding 'from Dockerfile' i stedet for nginx.conf?

Reply:
bra funn!